package.json

```diff
@@ -40,6 +40,7 @@
     "babel-preset-es2015": "6.16.0",
     "babel-preset-stage-2": "6.17.0",
     "eslint": "3.8.0",
+    "jest": "^17.0.3",
```

Comment:
Nit: I like to keep my dependencies exact to help reduce install issues. Can you update so this is exact?